content/docs/tasks/security/authz-http/index.md

```diff
@@ -70,6 +70,8 @@ for the list of supported keys in `constraints` and `properties`.
 
 ## Authorization permissive mode
 
+Note that Authorization permissive mode is an experimental feature in Istio release 1.1, its interface may change in future releases.
```

Comment:
Add: You could skip it and go directly to "Enable Istio Authorization"(link) if you do not want to try out permissive mode feature.

Reply:
done.